ui/tsconfig.json

```diff
@@ -24,7 +24,8 @@
     "checkJs": false,
     "sourceMap": true,
     "esModuleInterop": true,
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "resolveJsonModule": true
```

Comment:
like... shouldn't this have broken sooner?! the things that keep me up at night...